src/app/create/basicauction/Basicauction.css.ts

```diff
@@ -0,0 +1,132 @@
+import { style } from '@vanilla-extract/css';
+
+export const auctionForm = style({
+  margin: '10px',
+});
+
+export const auctionLabel = style({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '20px',
+  marginBottom: '30px',
+});
+
+export const title = style({
+  display: 'flex',
+  fontSize: '18px',
+  fontWeight: 'bold',
+});
+
+export const auctionInput = style({
+  width: '368px',
```

Comment:
```suggestion
  maxWidth: '368px',
```
이렇게 하면 페이지 줄어들면 사이즈도 바뀔 것 같아요우~